src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,74 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Status } from '../types/Status';
+
+type Props = {
+  filterBy: Status,
+  onFilterChange: (filterBy: Status) => void,
+  activeTodosCount: number,
+  hasCompletedTodosCount: boolean,
+  handleClearCompleted: () => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  filterBy,
+  onFilterChange,
+  activeTodosCount,
+  hasCompletedTodosCount,
+  handleClearCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeTodosCount} items left`}
```

Comment:
When there is only one todo there should not be plural in the message:
<img width="558" alt="image" src="https://github.com/mate-academy/react_todo-app-with-api/assets/54860641/fe415a2d-4dc7-419c-b2d3-65443407f149">